pkg/storaged/clevis-luks-passphrase.sh

```diff
@@ -30,10 +40,14 @@ if cryptsetup isLuks --type luks1 "$DEV"; then
 elif cryptsetup isLuks --type luks2 "$DEV"; then
     for id in `cryptsetup luksDump "$DEV" | sed -rn 's|^\s+([0-9]+): clevis|\1|p'`; do
         tok=`cryptsetup token export --token-id "$id" "$DEV"`
-        jwe=`jose fmt -j- -Og jwe -o- <<<"$tok" | jose jwe fmt -i- -c`
```

Comment:
`<<<` is a bashism, this replaces it with POSIX shell. This goes together with changing the hashbang "bash" to "sh". This could be its own independent commit to make it clearer. (FTR: There is no problem with using bash, this is just a stylistic decision)

Reply:
This is used with `cockpit.script` which always uses `/bin/sh`. Previously, the script would not run at all on Debian (because there is no `clevis` binary there). Now it runs always and thus the old version would fail on Debian.

There is no strong reason to run it also on Debian, but it also does no harm. And since I was changing it anyway, why not make it work with plain /bin/sh.

Reply:
There actually is a `clevis` package in Debian, let's test it!

Reply:
See #15835. I have moved the /bin/sh fixes to that PR.